otel/otel.go

```diff
@@ -0,0 +1,62 @@
+package otel
+
+import (
+	"context"
+	"github.com/armory-io/go-commons/metadata"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/fx"
+	"go.uber.org/zap"
+)
+
+type Configuration struct {
+	Endpoint string
+	APIKey   string
+}
+
+func NewTracerProviderProvider(
```

Comment:
How does this compare to what is going on here: 
https://github.com/armory-io/go-commons/blob/main/tracing/tracing.go